continuous-pipe.yml.erb

```diff
@@ -220,3 +248,17 @@ tasks:
               limits:
                 cpu: 1
                 memory: 1G
+
+  migration:
```

Comment:
personally I prefer in order of execution in this case though. The name doesn't hold much meaning